pubsub/jetstream/metadata.go

```diff
@@ -141,8 +141,8 @@ func parseMetadata(psm pubsub.Metadata) (metadata, error) {
 		m.rateLimit = v
 	}
 
-	if v, err := time.ParseDuration(psm.Properties["hearbeat"]); err == nil {
-		m.hearbeat = v
+	if v, err := time.ParseDuration(psm.Properties["heartbeat"]); err == nil {
```

Comment:
This is a breaking change however. Can you please:

1. Add a fallback logic so if the `hearbeat` property (with typo) is set, but not `heartbeat`, then `hearbeat` is used.
2. Please update the documentation for the component at https://github.com/dapr/docs too